tests/infrastructure/instance_types/test_common_vm_preference.py

```
@@ -52,10 +57,20 @@ def start_vm_with_cluster_preference(client, preference_name, namespace_name):
         running_vm(vm=vm, wait_for_interfaces=False, check_ssh_connectivity=False)
 
 
+def _preference_requires_efi(client, preference_name):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_preference_requires_efi`

(ANN202)